### PR TITLE
Fixed seeds data

### PR DESCRIPTION
**Pull request by coderdan**
Fixes SITES-645 but also raised SITES-646 as tech debt.

### CODE REVIEW

**Comment by bjgordon** [comment]
LGTM

File: app/models/section.rb [hunk 1]
```diff
@@ -46,10 +46,11 @@ def set_default_cms_type
 
   # FIXME: this should be done in a controller to break a circular dependency.
   # Fabricators cannot create a SectionHome which creates a Section which creates a SectionHome!
+  # Perhaps a better way is to use a Creator service object (DD)
   def generate_home_node
     unless Rails.env.test?  # <-- added this to avoid breaking specs
       unless home_node.present?
-        SectionHome.create! do |node|
+        create_home_node! do |node|
```
Comment (tessereth) [inline app/models/section.rb]: Where does `create_home_node!` come from?

Comment (coderdan) [inline app/models/section.rb]: http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_one

Comment (tessereth) [inline app/models/section.rb]: Huh. I learnt something today. Cool.

**Comment by tessereth** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

**Comment by coderdan** [comment]
Bump LGTM

**Pull request merged**